### PR TITLE
Pull request demo

### PR DESCRIPTION
**Pull request by pnchinmay**
- added another commit
- hello

### CODE REVIEW

**Comment by Thuglife1730** [comment]
iphone is better than android phone

**Review by Thuglife1730 — APPROVED** [review]
GG

File: A.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-Yeh Dil mange more
\ No newline at end of file
+Yeh Dil mange more
+Hello world
```
Comment (Thuglife1730) [inline A.txt:2]: change this to 
```suggestion
Pixel
```

**Review by Thuglife1730 — CHANGES_REQUESTED** [review]
Suggestion dekh lena

File: A.txt (hunk 1 above)
Comment (pnchinmay) [reply A.txt]: Committed those changes

File: A.txt [hunk 2]
```diff
@@ -1 +1,2 @@
-Yeh Dil mange more
\ No newline at end of file
+Yeh Dil mange more
+Pixel
```
Comment (Thuglife1730) [inline A.txt:2]:
```suggestion
Haale
YO HO
```

**Review by Thuglife1730 — CHANGES_REQUESTED** [review]
See the changes dear

File: A.txt (hunk 2 above)
Comment (pnchinmay) [reply A.txt:2]: Will send another commit

**Pull request merged**